pkg/sql/codegen/codegen_feature_column.go

```diff
@@ -39,9 +41,14 @@ func isXGBoostModule(module string) bool {
 	return strings.HasPrefix(module, "xgboost")
 }
 
-// IntArrayToJSONString converts int array to json string
-func IntArrayToJSONString(intArray []int) string {
-	return strings.Join(strings.Split(fmt.Sprint(intArray), " "), ",")
+// MarshalToJSONOrDie converts any data to JSON string.
+// Exit the process if there is any error.
+func MarshalToJSONOrDie(in interface{}) string {
+	bytes, err := json.Marshal(in)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Does this error end the sqlflowserver process? If so, this is not recommended since we don't need to stop the server process if there's an error occurred when processing one statement.

Reply:
Fix in https://github.com/sql-machine-learning/sqlflow/pull/2433 .